docs/_includes/common/filter-menu.html

```diff
@@ -0,0 +1,541 @@
+<div id="filter-menu">
+    <h2><span class="secondary-text">@ebay/skin/</span>filter-menu</h2>
+
+    <h3 id="filter-menu-aria">Filter Menu – ARIA Version – Default</h3>
+    <p>The ARIA version of the filter menu is a <em>statelful</em> menu. It's primary purpose is used in conjunction with <a href="#filter-menu-button">filter-menu-button</a>,
+        but can alternatively be used as a standalone menu when necessary.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="filter-menu-aria">
+                <form name="filter-menu-aria-1">
```

Comment:
I don't think a form tag is necessary for the ARIA version. There is no form data to post.

Reply:
This was a miss! (Copy pasta.) Let me remove.